test/views/UploadYourFilesSpec.scala

```diff
@@ -25,7 +25,9 @@ import views.html.upload_your_files
 
 class UploadYourFilesSpec extends DomAssertions with ViewBehaviours with PropertyChecks with Generators {
 
-  def view(pos: Position): Html = upload_your_files(UploadRequest("", Map.empty), pos)(fakeRequest, messages, appConfig, fakeRequest.flash)
+  val page = app.injector.instanceOf[upload_your_files]
```

Comment:
Just in general all the View Specs contains application in the trait. SpecBase that is in every test file contains injected messages, appConfig, mcc and created application. I used this just to make this appraoch closer to the correct one. If we will have more time we need to remove the application from all tests.

What is necessary in the view specs is create a main_template which we can pass to the view to create it instead of injecting.